src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -579,6 +580,9 @@ def __call__(
             num_inference_steps (`int`, *optional*, defaults to 50):
                 The number of denoising steps. More denoising steps usually lead to a higher quality image at the
                 expense of slower inference.
+            max_inference_steps (`int`, *optional*):
+                Instead of completing the backwards pass entirely, stop and return the output after this many steps.
```

Comment:
Let's maybe use an alterative term for "backwards pass" so that people don't confuse this with backdrop.

Reply:
I would also include the example you gave me here so that people get the idea immediately: https://github.com/huggingface/diffusers/issues/4003#issue-1795176432. 

WDYT?

Reply:
I hesitated because I wasn't sure how busy these can be. but I agree.